src/util/HistoryHijacker.ts

```diff
@@ -0,0 +1,33 @@
+export interface PopStateEvent {
+	state: string;
+}
+
+export interface HistoryHijacker {
```

Comment:
I don't understand the function of this interface from first glance and/or from the implementation. Could you add some docblocks that explains the pupose of the interface and what the methods do?